.pipelines/v2/templates/steps-esrp-sign-files-authenticode.yml

```diff
@@ -0,0 +1,45 @@
+parameters:
```

Comment:
This is a simple wrapper around ESRP signing for the kind of signing we do _most often_. It just makes the code shorter to not have the inline sign params 300 times